### PR TITLE
Music album

### PR DESCRIPTION
**Pull request by Amedzro-Elikplim**
- Create musicAlbum class
- Create genre class
- Create an association between genre and item class
- Implement methods in musicAlbum class
- Implement methods in genre class
- Create setters and getters for instance variables

### CODE REVIEW

File: app/genre.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+require_relative './item'
+
+class Genre
+  attr_accessor :items
+
+  def initialize(name)
+    @id = Random.rand(1..1000)
+    @name = name
+    @items = []
+  end
```
Comment (gedewon) [inline app/genre.rb]: image

as you see from the UML diagram. items is privately attribute and name is public so you might want to do something like this 
```rb
attr_acccessor :name
attr_reader :items

File: app/music.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+  def initialize(on_spotify)
+    @on_spotify = on_spotify
+  end
+
+  def can_be_archived?
+    true if super() && @on_spotify == true
+    false
+  end
+end
```
Comment (gedewon) [inline app/music.rb]: MusicAlbum has an inheritance relationship with Item class please include those.

**Review by gedewon — CHANGES_REQUESTED** [review]
Hi @Amedzro-Elikplim , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 


## Required Changes ♻️
_Check the comments under the review._

File: app/music.rb (hunk 2 above)
Comment (Amedzro-Elikplim) [reply app/music.rb]: Create Genre class with an association to the Item class (in a separate .rb file).
MusicAlbum has no inheritance association with item class.....or am i missing something

Comment (Amedzro-Elikplim) [reply app/music.rb]: i get it now

**Pull request merged**